scripts/install.sh

```diff
@@ -0,0 +1,18 @@
+#install the libraries below
+arduino-cli lib install Adafruit_Keypad@1.0.4
+arduino-cli lib install Adafruit_MAX31865@1.0.3 #Has a duplicate maybe? May need a double check
+arduino-cli lib install PID@1.2.1
+arduino-cli lib install ArduinoJson@6.14.1
+arduino-cli lib install AsyncTaskLib@1.0.0
+arduino-cli lib install Ethernet@2.0.0
+arduino-cli lib install Keypad@3.1.1
+arduino-cli lib install LiquidCrystal@1.0.7
+arduino-cli lib install Marceau@0.0.1
+arduino-cli lib install MemoryFree@latest #Special, hosted locally with no properties page
+arduino-cli lib install PID_AutoTune_v0@0.0.1 #Has no dedicated properties page
```

Comment:
Remove PID_AutoTune_v0 as it does not appear to be in the Arduino library. We can look at adding items to the script that download the github code directory. https://github.com/br3ttb/Arduino-PID-AutoTune-Library